tsl/src/compression/compression.c

```diff
@@ -83,6 +83,21 @@ DecompressionIterator *(*tsl_get_decompression_iterator_init(CompressionAlgorith
 		return definitions[algorithm].iterator_init_forward;
 }
 
+ArrowArray *
+tsl_try_decompress_all(CompressionAlgorithms algorithm, Datum compressed_data, Oid element_type)
```

Comment:
We are calling `decompress_all_forward_direction` in this function. Is it planned to implement this also for the backward direction in a follow-up PR? If so, I suggest renaming the function (e.g. to`tsl_try_decompress_all_forward_direction`).